static/app/views/insights/common/components/tableCells/starredSegmentCell.tsx

```diff
@@ -14,8 +14,10 @@ interface Props {
   segmentName: string;
 }
 
-type TableRow = Partial<EAPSpanResponse> &
-  Pick<EAPSpanResponse, 'is_starred_transaction' | 'transaction'>;
+interface TableRow extends Partial<EAPSpanResponse> {
+  is_starred_transaction: boolean;
+  transaction: string;
+}
```

Comment:
This makes sense, but icky to re-declare `is_starred_transaction` and `transaction` types here just to express that they're not optional. What do you think about

```jsx
type TableRow = Flatten<Partial<EAPSpanResponse> &
  Pick<EAPSpanResponse, 'is_starred_transaction' | 'transaction'>>;
```